portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,18 +50,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         for (Entity entity: results.asIterable()) {
             long id = entity.getKey().getId();
             String name = (String) entity.getProperty("name");
-            String email = (String) entity.getProperty("email");
             String comment = (String) entity.getProperty("comment");
+            String email = (String) entity.getProperty("email");
+
             Date date = (Date) entity.getProperty("date");
-            UserData newData = new UserData(id, name, email, comment, date);
+            UserData newData = new UserData(id, name, comment, date, email);
```

Comment:
do you want to return the user's email? in many websites this would be considered privacy breaking, since you are basically allowing any website scraper unrestricted access to personal information of your users...

Reply:
I don't think the email is necessary here, so i took it out. I kept the emails in the data-store tho